baselines/fedpara/README.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "FedPara: Low-rank Hadamard Product for Communication-Efficient Federated Learning"
+url:  https://openreview.net/forum?id=d71n4ftoCBy
+labels: [image classification, personalization, low-rank training, tensor decomposition]
+dataset: [CIFAR-10, CIFAR-100, FEMNIST]
```

Comment:
```suggestion
dataset: [CIFAR-10, CIFAR-100, MNIST]
```

Reply:
please update the readme indicating that the personalisation dataset used is MNIST.